synapse/events/utils.py

```diff
@@ -487,15 +486,22 @@ def _inject_bundled_aggregations(
         """
         serialized_aggregations = {}
```

Comment:
this is a bit out of place now, move down to after `event_aggregations = bundled_aggregations[event.event_id]` ?